src/main.h

```diff
@@ -133,8 +134,13 @@ static const uint32_t MAX_UNCONNECTED_HEADERS = 144;
 /** The maximum length of time, in seconds, we keep unconnected headers in the cache **/
 static const uint32_t UNCONNECTED_HEADERS_TIMEOUT = 120;
 
-static const unsigned int DEFAULT_LIMITFREERELAY = 15;
-static const bool DEFAULT_RELAYPRIORITY = true;
+/** The maximum number of free transactions (in KB) that can enter the mempool per minute.
+ *  For a 1MB block we allow 15KB of free transactions per 1 minute.
+ */
+static const uint32_t DEFAULT_LIMITFREERELAY = DEFAULT_BLOCK_MAX_SIZE * 0.000015;
+/** Subject free transactions to priority checking when entering the mempool */
+static const bool DEFAULT_RELAYPRIORITY = false;
```

Comment:
what's the reason we need change it?

Reply:
Other implementations don't have this anymore and we already have two other levels of spam control.  Also the filtering by priority is totally arbitrary...the calculation for the priority cutoff is defined as : inline double AllowFreeThreshold() { return COIN * 144 / 250; }

It's just a number somebody picked...if you're priority is below the threshold then you're not getting into the memory pool and your txn isn't getting fowarded.  Seems totally arbitrary to me and goes against the BU and BitcoinCash philosophy of allowing more free transactions into the network.

Reply:
Agreed on the fact that the current prio threshold is pretty arbitrary.
AFAIK Bitcoin ABC still have `DEFAULT_RELAYPRIORITY` set to `false` and rely on the same value for the prio thres.
Regardless I can see it as a valuable change.